lib/js/components/font-loading.mjs

```diff
@@ -131,16 +131,18 @@ export class FontSelect extends _BaseComponent {
           , optGroups = {}
           , createOptGroup = name=>{
                 const optgroup = this._domTool.createElement('optgroup');
+                const legend = this._domTool.createElement('legend');
                 switch(name) {
                     case 'from-url':
-                        optgroup.label ='Included fonts';
+                        legend.innerHTML ='Included fonts';
```

Comment:
Please use `legend.textContent` rather than `innerHTML` in all cases. There's no reason to involve HTML parsing and I prefer HTML parsing done explicitly. `textContent` can cause less undesired effects, e.g. in cases when user data is provided.